src/py21cmfast/wrapper.py

```diff
@@ -1734,6 +1741,9 @@ def run_coeval(
                     "PID={} doing brightness temp for z={}".format(os.getpid(), z)
                 )
                 ib_tracker[redshift.index(z)] = ib2
+                if flag_options.USE_TS_FLUCT:
```

Comment:
Can't this just be a single line with `st_tracker[redshift.index(z)] = st2 if flag_options.USE_TS_FLUCT else None`

Reply:
Yes! Did that.